### DISCUSSION TITLE
Static Hostnames - Stamp to short

### DISCUSSION THREAD

**Discussion by TheCherry** [DNSCrypt/dnscrypt-proxy#1777] [Q&A]
Hi,

I want configure some static hostnames for my local network. My plan is to use dnscrypt-proxy for the entire network, so I let it run on a server in a docker container. Now I want that all *.local requests response a local network address. I tried to setup this with the [static] method, but that seems only work for a directly DNS Server am I right? I tried this method anyway and configure just the IP 192.168.1.2 with https://dnscrypt.info/stamps/  and added it with:
[static.'*.local']
stamp = 'sdns://AQcAAAAAAAAACzE5Mi4xNjguMS4yABAyLmRuc2NyeXB0LWNlcnQu'

Then I get the error message on start:
Stamp to short.

Is there no way to define directly hostnames? Do I really have to setup a extra DNS Server for this?

**Comment by jedisct1** [comment]
Looking at the documentation, it seems that [cloaking](https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Cloaking) is the mechanism design to directly provide a map from names to IP addresses.

The following page also seems to be related, describing how to [forward](https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Forwarding) domains to the local router.
